routes/api.js

```diff
@@ -158,6 +159,66 @@ router.patch('/user/badge/update', authenticationHelpers.isAuthOrRedirect, (req,
     }
 });
 
+router.post('/user/activity/level', authenticationHelpers.isAuthOrRedirect, (req, res) => {
+    if (!req.user.email) {
+        return res.status(400).json({error: errorCode.PCE021.message, code: errorCode.PCE021.code});
+    }
+    localUser.find({where: {
+        email: req.user.email
+    }}, {raw: true})
+        .then(data => {
+            if (!data) {
+                return res.status(200).json({info: errorCode.PCE022.message, code: errorCode.PCE022.code});
```

Comment:
status 200 is for success messages.  You should be using 400

Reply:
Previous PR.

Reply:
I believe 200 status is correct, here the request went through but there was no data available. 400 is for bad request, which means the request was malformed or had incorrect parameters.

Reply:
Oops ! 
Reading a little this is what I found,

4xx: Client Error - It means the request contains incorrect syntax or cannot be fulfilled.
5xx: Server Error - It means the server failed to fulfill an apparently valid request.

So should it be 500 @Raxerz  because 200 means success.

Reply:
@shreyans29 
https://stackoverflow.com/questions/38659310/what-http-return-code-should-be-if-no-data-available

The thread here says 200 is the correct status code to be sent for such cases.